### PR TITLE
ci: update Travis CI test suite to gonum/gonum standards

### PR DESCRIPTION
**Pull request by sbinet**
Fixes gonum/exp#18.

### CODE REVIEW

**Comment by sbinet** [comment]
PTAL.

File: .travis/test-coverage.sh [hunk 1]
```diff
@@ -27,8 +27,8 @@ testCover() {
 # Init acc.out
 echo "mode: set" > $ACC_OUT
 
-# Run test coverage on all directories containing go files except testlapack and testblas.
-find . -type d -not -path '*testlapack*' -and -not -path '*testblas*' | while read d; do testCover $d || exit; done
+# Run test coverage on all directories containing go files except testlapack testblas and testgraph.
```
Comment (kortschak) [inline .travis/test-coverage.sh]: These are presumably not important here.

**Comment by sbinet** [comment]
PTAL

**Comment by kortschak** [comment]
Oh, sorry! I forgot about this one.

LGTM after conflicts are fixed.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+# Gonum exp [![Build Status](https://travis-ci.org/gonum/exp.svg?branch=master)](https://travis-ci.org/gonum/exp) [![Build status](https://ci.appveyor.com/api/projects/status/8aj3pwwukp1n1u09/branch/master?svg=true)](https://ci.appveyor.com/project/Gonum/exp/branch/master) [![stability-experimental](https://img.shields.io/badge/stability-experimental-orange.svg)](https://github.com/emersion/stability-badges#experimental) [![Coverage Status](https://coveralls.io/repos/gonum/exp/badge.svg?branch=master&service=github)](https://coveralls.io/github/gonum/exp?branch=master) [![GoDoc](https://godoc.org/gonum.org/v1/exp?status.svg)](https://godoc.org/gonum.org/v1/exp) [![Go Report Card](https://goreportcard.com/badge/github.com/gonum/exp)](https://goreportcard.com/report/github.com/gonum/exp)
```
Comment (kortschak) [inline README.md]: On the gonum/gonum README we have the stability badge at the end. I don't know if that's the best order, but we should probably be consistent.

**Comment by sbinet** [comment]
should I squash? or rebase+merge?
(I should probably reword a bit the last commit `s/reorder stability badge/ci: reorder stability badge/`)

**Comment by kortschak** [comment]
I think a rebase is fine.

**Pull request merged**